packages/reports/addon/components/navi-column-config/item.js

```diff
@@ -69,12 +69,20 @@ class NaviColumnConfigItemComponent extends Component {
   }
 
   /**
-   * Stores element reference after render
+   * Stores element reference after render.
+   * If the column was just added, open the config and scroll to element
    * @param element - element inserted
    */
   @action
   setupElement(element) {
     this.componentElement = element;
+
+    if (this.isLastAdded) {
+      set(this, 'isColumnConfigOpen', true);
+      next(() => {
+        element.parentElement.scrollTop = element.offsetTop - element.parentElement.offsetTop;
```

Comment:
Found [scrollIntoView](https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoView) which seems a little cleaner, but either way is fine

```suggestion
        this.componentElement.scrollIntoView(true);
```

Reply:
yeah much better, I didn't like the component scrolling its parent

Reply:
so I just noticed the scroll doesn't work if the drawer is closed before adding :/